src/components/Editor.js

```diff
@@ -382,6 +383,45 @@ const Editor = React.createClass({
     showMenu(e, items);
   },
 
+  traverseResultsPrev(e, ctx) {
+    e.stopPropagation();
+    e.preventDefault();
+    const { query } = this.state;
+    findPrev(ctx, query);
+  },
+
+  traverseResultsNext(e, ctx) {
+    e.stopPropagation();
+    e.preventDefault();
+    const { query } = this.state;
+    findNext(ctx, query);
+  },
```

Comment:
hmm, i think these function could receive query and be moved to helpers above the component.

My rationale is that we already have a lot of instance functions :)